packages/react/src/FormErrorList/FormErrorList.tsx

```diff
@@ -25,6 +26,8 @@ export type FormErrorListProps = {
    * Note: this intentionally does not change the font size.
    */
   headingLevel?: HeadingLevel
+  /** The text following the error count. */
+  docTitleSuffix?: { plural: string; singular: string }
```

Comment:
It’s not the suffix of the document title. How about `errorText`, `errorLabel`, `errorTerm`? Maybe even `errorCount*` to match the hook’s name.

Reply:
Isn't `error*` a little confusing? This is a prop for `FormErrorList`, which of itself doesn't have anything to do with the document title. Specifying that this prop is about the document title feels necessary imo. How about `docTitleLabel`?

Reply:
I proposed `error*` from an opposite perspective: letting the component ask for the singular and plural forms of the noun – without disclosing why it needs them, on purpose.

If we consider setting the title as an implementation detail, we can maintain a more generic content model for this component: “I am responsible for listing errors of a form. Please provide me with a heading and its level, the list of errors, and the words to use for ‘error’ and ‘errors’; and I will take care of everything.” The documentation lets people know that the component updates the document title next to rendering a box in the page.

When coupling the words with the document label, we would need a second prop with the same content (or rename it) if the component needs to render these words elsewhere later (visually, audibly…).

I’d prefer thinking from the content the component needs, not where it needs them for.

Reply:
We generally try to use prop names that give the user an indication of what they should enter there, no? A `heading` prop will render the `heading` of a component, so you should enter the heading text there, etc. 

Something like `errorText` does not communicate the same, imo. I'd have to read the docs to know what I'd have to enter there. I'm not sure `docTitleLabel` is the way to go here, but it gives the user a better indication of what they should enter there. 

> When coupling the words with the document label, we would need a second prop with the same content (or rename it) if the component needs to render these words elsewhere later (visually, audibly…).

I don't think we should look ahead too much. For the foreseeable future, this prop is used to set a part of the doc title, so that can be reflected in the name imo.

Reply:
> We generally try to use prop names that give the user an indication of what they should enter there, no?

Of course. But a name that describes a field of a thing clearly, without mentioning where it appears, does meet that objective.

> Edit: Thought about it some more, maybe `errorCountLabel` together with a JSDoc description that it's used in the doc title is sufficient?

Yes, I can totally go along with that name.

Reply:
Allrighty, updated it!